apps/web/src/ee/translations/components/CreateGroupSidebar.tsx

```diff
@@ -116,7 +116,7 @@ export const CreateGroupSidebar = ({
       }
     >
       <FormProvider {...methods}>
-        <GroupFormCommonFields control={control} readonly={readonly} />
+        <GroupFormCommonFields control={control as Control<ICreateGroup>} readonly={readonly} />
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** I'm not sure why this appeared now, but my build kept failing locally without this.‏